### PR TITLE
add prompts

### PR DESCRIPTION
**Pull request by lemon-awa**
Update few-shot prompts for evaluating the five aspects of summarization (content, key-idea, method, outcome, future-impact). The purpose of prompts is giving a similarity score on a scale from 1 to 10 for two given inputs.These prompts examples are half by hand-crafted half from real examples.
**Overall Workflow：** 
1. An article is selected at random. And all its summarizations both by human annotators and llm have been collected.
2. For each aspect, manually annotated examples deemed the most comprehensive and accurate are chosen as targets (Input 1 in each prompt).
3. Key phrases or keywords (4-5) are extracted from the target sentences to serve as main evaluation criteria. For example, in the `context` prompt, selected keywords could be "Non-pharmacological interventions," "people with dementia," "biographical or personalized materials," "benefit these interventions," and "challenging to obtain and maintain.".
4. Scores can be divided into 5 segments: 1-2, 3-4, 5-6, 7-8, 9-10. The degree of similarity between Input 2 and the selected keywords from input1 determines the output score segment. The logical coherence and sentence structure of Input 2 are also evaluated, with a margin of 1 point for bias.
5. Example Scoring Justification:
    -   Typically example with 9-10 scores are sourced from another annotator's summary.
    -   Example with 1-2 scores are from summaries of different aspects of the same essay.
    -   Other examples come from modification of LLM-generated summarizations, edited to align with the score segment.
    -   Justifications for each score are the real response of the prompt by calling openai API, with slight modifications to ensure consistent formatting and criteria alignment.

### CODE REVIEW

**Comment by lemon-awa** [comment]
Update few-shot prompts for evaluating the five aspects of summarization (content, key-idea, method, outcome, future-impact). The purpose of prompts is giving a similarity score on a scale from 1 to 10 for two given inputs.These prompts examples are half by hand-crafted half from real examples.
**Overall Workflow：** 
1. An article is selected at random. And all its summarizations both by human annotators and llm have been collected.
2. For each aspect, manually annotated examples deemed the most comprehensive and accurate are chosen as targets (Input 1 in each prompt).
3. Key phrases or keywords (4-5) are extracted from the target sentences to serve as main evaluation criteria. For example, in the `context` prompt, selected keywords could be "Non-pharmacological interventions," "people with dementia," "biographical or personalized materials," "benefit these interventions," and "challenging to obtain and maintain.".
4. Scores can be divided into 5 segments: 1-2, 3-4, 5-6, 7-8, 9-10. The degree of similarity between Input 2 and the selected keywords from input1 determines the output score segment. The logical coherence and sentence structure of Input 2 are also evaluated, with a margin of 1 point for bias.
5. Example Scoring Justification:
    -   Typically example with 9-10 scores are sourced from another annotator's summary.
    -   Example with 1-2 scores are from summaries of different aspects of the same essay.
    -   Other examples come from modification of LLM-generated summarizations, edited to align with the score segment.
    -   Justifications for each score are the real response of the prompt by calling openai API, with slight modifications to ensure consistent formatting and criteria alignment.

File: prompts/context.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    },
+    {
+        "role": "assistant",
+        "content": "{\n \"score\": 10,\n \"justification\":Both inputs discuss the effectiveness of non-pharmacological interventions for people with dementia, specifically mentioning the use of biographical or personalized materials. They also both highlight the challenge of obtaining and maintaining such information. The semantic content is nearly identical lead to score of 10. \"\" \n}"
+    },
+    {
+        "role": "user",
+        "content": "Input 1:Non-pharmacological interventions are the most common and the most effective for people with dementia.  Using biographical or personalized materials has been proven to benefit these interventions, but it can be challenging to obtain and maintain such information.\nInput 2:Some interventions that effective for people with dementia have been proven to benefit from the usage of biographical or personalized materials, which are not always easy to obtain and challenging to maintain."
```
Comment (xingjian-zhang) [inline prompts/context.json:16]: Grammar mistake in Input 2.

File: prompts/key_idea.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        "content": "{\n \"score\": 8,\n \"justification\"::Both inputs describe a web platform aimed at supporting the care of people with dementia by streamlining the collection of relevant information.   While input 2 miss one key-point: the platform is supporting psychologists generally, leading to score of 8. \"\" \n}"
+    },
+    {
+        "role": "user",
+        "content": "Input 1:The authors create a web platform that supports the work of psychologists, streamlining the collection of relevant information about people with dementia.\nInput 2:The authors created a web platform to support psychologists by streamlining the collection of relevant biographical information about people with dementia, facilitating reminiscence and other biographical cognitive stimulation practices."
+    },
+    {
+        "role": "assistant",
+        "content": "{\n \"score\": 10,\n \"justification\":Both inputs describe a web platform designed to assist psychologists by streamlining the collection of information about people with dementia.   Input 2 provides a little additional detail by mentioning biographical information, but the core idea remains nearly identical. \"\" \n}"
```
Comment (xingjian-zhang) [inline prompts/key_idea.json:44]: Input 2 clearly contains more details than Input 1. I think this is not a perfect 10 (considering your example in context.json)

**Review by xingjian-zhang — COMMENTED** [review]
Thanks @lemon-awa. The examples now look better. A few suggestions:
- Could you please reorder the prompts so that the scores are ascending or descending for every file?
- Could you place the justification prior to score? This might help LLM to give cot-like reasoning before giving a score.
- Let's also update the corresponding code to load these prompts. Specifically the `LLMSimilarity` class.

**Review by xingjian-zhang — APPROVED** [review]

**Pull request merged**